api/v1alpha1/openapi.yaml

```
@@ -4121,6 +4177,16 @@ components:
         - "DeviceResourceStatusCritical"
         - "DeviceResourceStatusError"
         - "DeviceResourceStatusUnknown"
+    DeviceLabelMap:
+      type: object
+      additionalProperties:
+        type: string
+      description: |
+        A map of distinct labels, where the keys are label names and the values are label values.
```

```suggestion
        A map of distinct labels, where the keys are label keys and the values are label values.
```
which is quite redundant

As this likely will change, let's see what the best description for the new response.